src/test/java/uk/gov/hmcts/reform/jobscheduler/services/jobs/jobsservice/GetAllJobTest.java

```diff
@@ -104,9 +110,22 @@ public void should_return_a_list_of_jobs_correctly_paginated() throws SchedulerE
                         .withDescription("description")
                         .usingJobData(jobDataMap)
                         .build();
+                    Trigger trigger = newTrigger()
```

Comment:
`setUpAndRetrieve` method has grown very long. I think it's the right moment to extract some bits out of it.

Also, this trigger has been added but tests didn't change. We should have a test inspecting the content of a job, including the trigger.